issues.go

```diff
@@ -317,6 +317,7 @@ type UpdateIssueOptions struct {
 	UpdatedAt        *time.Time `url:"updated_at,omitempty" json:"updated_at,omitempty"`
 	DueDate          *ISOTime   `url:"due_date,omitempty" json:"due_date,omitempty"`
 	DiscussionLocked *bool      `url:"discussion_locked,omitempty" json:"discussion_locked,omitempty"`
+	Weight           *int       `url:"weight,omitempty" json:"weight,omitempty"`
```

Comment:
Please keep the same order as used in the docs so fields are easy to find. Thanks!